demo-app/src/pages/ot/config-builder/index.tsx

```diff
@@ -31,9 +31,12 @@ const Output: React.FC = () => {
       {!showConfig && (
         <RenderConfig
           config={config}
-          components={Object.assign(OT_COMPONENTS, {
-            ConfigBuilderFrameWrapper,
-          })}
+          components={Object.assign(
+            { ...OT_COMPONENTS },
+            {
+              ConfigBuilderFrameWrapper,
+            },
```

Comment:
👍 this works. Fwiw I've always seen `Object.assign` used with an empty object as it's first arg since it mutates it, but your way is a bit shorter.

```javascript
> const a = { a: 1 }
undefined
> const b = { b: 2 }
undefined
> Object.assign({}, a, b)
{ a: 1, b: 2 }
> Object.assign({ a: 3 }, a, b)
{ a: 1, b: 2 }
```